book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Please create `add_rental` method in book class that will accept date and person as parameters to create rentals and for the book assign self.